Tests/MuxUploadSDKTests/Upload Tests/UploadPersistenceTests.swift

```diff
@@ -62,14 +65,24 @@ final class UploadPersistenceTests: XCTestCase {
             uploadInfo: renameDummyUploadInfo(basename: "e2")
         )
         let persistence = UploadPersistence(innerFile: makeSimulatedUploadsFile(), atURL: makeDummyFileURL(basename: "fake-cache-file"))
-        // Shouldn't throw (but not specifically part of the test)
-        try! persistence.write(entry: e1, forFileAt: makeDummyFileURL(basename: "e1"))
-        try! persistence.write(entry: e2, forFileAt: makeDummyFileURL(basename: "e2"))
+        XCTAssertNoThrow(
```

Comment:
`XCTAssertNoThrow` is effectively the same as `try!` (the test will fail and throw either way if this expression throws). One bonus behind asserting here is it does also allow us to pass a string that gets printed when a specific assertion fails.